### PR TITLE
chore(ckbtc): Propose to reduce retrieve_btc_min_amount for ckBTC minter

### PR DESCRIPTION
**Pull request by ninegua**
XC-233: ckBTC minter: Proposal to change min withdrawal amount

### CODE REVIEW

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Proposal to upgrade the ckBTC minter canister and reduce minimum retrieval amount
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `bcbccf79c89a3e81b1a38d8233f8f81d1af1b245`
+
+New compressed Wasm hash: `e90d3f24a6eb3e1da8f42f3160756c40598e2a6783097db1beeb31873224e787`
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md:7]: Compressed wasm hash confirmed 🥳

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The previous proposal https://dashboard.internetcomputer.org/proposal/133840 failed to change this setting because the ckBTC minter implementation was using a hardcoded value.
+A fix has since been implemented, and this upgrade will apply the fix.
+
+## Upgrade args
+
+```
+git fetch
+git checkout bcbccf79c89a3e81b1a38d8233f8f81d1af1b245
+cd rs/bitcoin/ckbtc/minter
+didc encode -d ckbtc_minter.did -t '(MinterArg)' '(variant {Upgrade = opt record { retrieve_btc_min_amount = opt 50000 }})' | xxd -r -p | sha256sum
```
Comment (gregorydemay) [inline rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md:29]: Should be (same as [previous proposal](https://dashboard.internetcomputer.org/proposal/133840))
`0b7d4ab8fe7adf7c609c7d565765abfa309ba7d03660439ec12c2e08e33ee6a5`

**Review by gregorydemay — APPROVED** [review]

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md (hunk 2 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md:29]: I got the same.

File: rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md (hunk 1 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/minter_upgrade_2024_11_13.md:7]: Confirmed!

**Review by THLO — APPROVED** [review]

**Pull request merged**